src/AgentCommerce/Exception/HoneyWebhookExceptions.php

```diff
@@ -0,0 +1,92 @@
+<?php declare(strict_types=1);
+/*
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Swag\PayPal\AgentCommerce\Exception;
+
+use GuzzleHttp\Exception\ClientException;
+use Shopware\Core\Framework\HttpException;
+use Shopware\Core\Framework\Log\Package;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @internal
+ */
+#[Package('checkout')]
+class HoneyWebhookExceptions extends HttpException
```

Comment:
its always only one:
```suggestion
class HoneyWebhookException extends HttpException
```